pkg/helm/presets/backend/high-resources.yaml

```diff
@@ -4,9 +4,6 @@ k8sWatcher:
     requests:
       cpu: 50m
```

Comment:
?

Reply:
@steb-veksler even we do we don't overwrite here, I still think it is better to have the values here explicit